docs/admin/access.rst

```diff
@@ -67,7 +67,7 @@ The default value can be changed by :setting:`DEFAULT_ACCESS_CONTROL`.
 .. warning::
 
     By turning on `Custom` access control, Weblate will remove all
-    :ref:`special groups <manage-acl>` it has created for this project.
+    :ref:`special groups <manage-acl>` it has created for selected project.
```

Comment:
_a_ selected project?

Reply:
Yes, thanks. Didn’t see it in the long run.

Reply:
Sorry, my error.

Yes, selected=the project you are in.

Reply:
solved